### DISCUSSION TITLE
안녕하세요 지금잘되고잇는건지몰겟네요.

### DISCUSSION THREAD

**Discussion by bear2898** [changdoc/naver-vaccine-macro#138] [Q&A]
![캡처](https://user-images.githubusercontent.com/88126712/127432766-e507eb09-8c7c-44c9-ae5c-b5970857916d.JPG)

지금 예약신청칸에서 새로고침 자기가알아서 대고 잇는거면 잘작동하고잇는건가요??

1)그리고 혹시 예약이 잡히면 예약신청은 제가눌러야하나요. 아님 자동으로 예약신청까지되는건가요?? 

2)예약신청 여러링크해서 여러군대 돌리는건 지금 안된다고공고봣는데 그게맞나요?ㅎㅎ

하 진짜 이런건 쥐약이라 너무어렵네요 .ㅠㅠ

**Comment by covid19bye** [comment]
백신 뜨면 자동으로 예약신청까지 된다고 합니다!
